egs/europarl_st/SRT/local/org_to_jsonl.py

```
@@ -0,0 +1,231 @@
+                )
+
+                segments_timestamps = read_lst_file(segments_lst_file)
+                segments_source_lang_transcriptions = read_lst_file(
+                    segments_source_lang_file
+                )
+                segments_dest_lang_transcriptions = read_lst_file(
+                    segments_dest_lang_file
+                )
+
+                segments_source_lang_transcriptions_dict = dict(
+                    zip(segments_timestamps, segments_source_lang_transcriptions)
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)